Removed Past Events

AlexCano21 opened the pull request:
1. Set the drafts = true, on the markdown files of past expanded learning events. They will not be visible to anyone except on the coding side.

ipnicholson commented:
@AlexCano21 I think the intent of this ticket was to update the content for these events, and not remove them entirely. For some, it could mean putting the wording into the past tense, and removing signup links. It's still nice to show past events on the website

AlexCano21 commented:
1. Changed all the descriptions to past tense 
2. Deleted the instructors, date and times, and the sign up links 
3. Added a "Past Event" label to the subtitle
